tests/Crypto/Polyfill/AesGcmTest.php

```diff
@@ -2618,9 +2618,8 @@ public function testCompat()
             $this->markTestSkipped('This test can only run on PHP 7.1');
             return;
         }
-        $ptLen = \random_int(0, 1024);
-        $aadLen = \random_int(0, 1024);
-        $i = 0;
+        $ptLen = \random_int(1, 1024);
+        $aadLen = \random_int(1, 1024);
```

Comment:
The reason the pipeline for this MR previously failed, was because `$aadLen` was 0, resulting in a failure when doing `$aad = \random_bytes($aadLen + $i);`.

This is now avoided by having the minimum-value of both lengths to be 1.